rotkehlchen/data/all_assets.json

```diff
@@ -15008,5 +15008,96 @@
         "started": 1608242396,
         "symbol": "LDO",
         "type": "ethereum token"
+    },
+    "BETH": {
```

Comment:
Remove this and just add a mapping in the binance exchange to ETH2. Beacon ETH is essentially ETH2, just from inside the binance exchange.